### PR TITLE
Fix/edb add repr method

### PR DESCRIPTION
**Pull request by dcrawforAtAnsys**
Updated docstring for Edb to demonstrate ``__getitem__()`` and ``__setitem__()``. 
Add ``__repr__()`` to pyaedt.edb_core.edb_data.EdbValue.

### CODE REVIEW

File: pyaedt/edb.py [hunk 1]
```diff
@@ -196,6 +203,29 @@ def __exit__(self, ex_type, ex_value, ex_traceback):
         if ex_type:
             self.edb_exception(ex_value, ex_traceback)
 
+    @pyaedt_function_handler()
+    def __getitem__(self, variable_name):
+        """Get or Set a variable to the Edb project. The variable can be project using ``$`` prefix or design variable.
+
+        Parameters
+        ----------
+        variable_name
```
Comment (MaxJPRey) [inline pyaedt/edb.py:212]:
```suggestion
        variable_name : str
```

File: pyaedt/edb.py [hunk 2]
```diff
@@ -1958,9 +1988,56 @@ def export_siwave_dc_results(
...
         )
 
+    @pyaedt_function_handler()
+    def variable_exists(self, variable_name):
+        """Check if a variable exists or not.
+
+        Returns
+        -------
+        tuple of bool and VaribleServer
+            It returns a booleand to check if the variable exists and the variable
```
Comment (MaxJPRey) [inline pyaedt/edb.py:1998]:
```suggestion
            It returns a boolean to check if the variable exists and the variable
```

File: pyaedt/edb.py [hunk 3]
```diff
@@ -1958,9 +1988,56 @@ def export_siwave_dc_results(
...
+            return True, var_server
+        return False, var_server
+
+    @pyaedt_function_handler()
+    def get_variable(self, variable_name):
+        """Return Variable Value if variable exists.
+
+        Parameters
+        ----------
+        variable_name
```
Comment (MaxJPRey) [inline pyaedt/edb.py:2022]:
```suggestion
        variable_name : str
```

**Review by svandenb-dev — APPROVED** [review]

**Comment by jsalant22** [comment]
@dcrawforAtAnsys @MaxJPRey This PR modified a bunch of EMIT files. It primarily looks like it reverted one or two of the more recent PRs from our team. Was this intentional or did something with the merge get screwed up?

**Pull request merged**